Automattic/jetpack
# Jetpack CRM: Remove WorDBless post-install-cmd and post-update-cmd

## Proposed changes:

These were left behind by PR #32498, which removed use of WorDBless.

### Other information:


## Jetpack product discussion


None

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Run `tools/composer-update-monorepo.sh projects/plugins/crm/`. There should be no complaint about "Class WorDBless\Composer\InstallDropin is not autoloadable, can not call post-update-cmd script"
